### DISCUSSION TITLE
关于共享值在合田之前设置一个阶段性封顶值的思考。

### DISCUSSION THREAD

**Discussion by famland16816** [famland/website#139] [Ideas]
建议共享值靠野田投入共享金额度可提升的共享值最高为500。
也就是共享值达到1000的时候，要想再提升必须通过合田。

一个基本田的投入是500U，增发量是693.5。
共享值的提升是，基本田每收取价值1U的YL。共享值可提升1。

那么收取价值达到500U的时候。相当于已经把他创造的增发量，领取完了。
但这个时候他的收益并没有停止。
获得的是，后种的田创造的增发量。

这个时候你这块田就不能提升共享值了。

要么等后面的田共享值，升起来赶上你。
要么，复投，建一块新田，提升新田的共享值。
正常情况下都会选择复投。
那么，你前面那块田，继续收取的相当是你复投的这块儿田创造的增量。

那么这个就是一个正向循环。

这个相当于一个，阶段性的封顶。
如果在合田之前没有一个封顶。
那么这个共享值的差距会越来越大。
越晚做市的，越难追上来。
即便用合田可以追上来，那要花相当大的成本，不知要合多少块田才能，赶上来。

设置这样一个阶段性封顶，可以减少这个差距，或者说减缓这个差距。

**Comment by yancanlou** [comment]
合田成功了基本上是本金打对折，单田收益又不能升两倍。不如躺着收两块的收益，只要共享值在又不是没有收益不是？
